locations/json_blob_spider.py

```diff
@@ -20,6 +20,9 @@ class JSONBlobSpider(Spider):
     # If set then use as a key into the JSON response to return the location data array.
     locations_key: str = None
 
+    # For passing to the subclass when iterating a JSON dict.
+    ATP_FEATURE_KEY = "atp:feature_key"
```

Comment:
It took me a while to figure out what was going on here, so here's an attempt at making the doc clearer.

This variable could also go outside of the class, since it's not an instance variable.

```suggestion
    # When processing a dict of locations in a JSON blob, the dictionary key of the scraped data will get
    # added to the item using this key so that DictParser uses it as a `ref` and the items get deduped
    # correctly.
    ATP_FEATURE_KEY = "atp:feature_key"
```